openam-shared/src/main/java/com/sun/identity/shared/debug/DebugConstants.java

```diff
@@ -42,7 +43,15 @@ public final class DebugConstants {
 
     public static final String CONFIG_DEBUG_LOGFILE_MAX_SIZE = "org.forgerock.openam.debug.rotation.maxsize";
 
-    public static final String DEFAULT_DEBUG_SUFFIX_FORMAT = "-yyyy.MM.dd-HH.mm.ss";
+    /**
+     * Control the type of formatter created by the debug formatter factory.
+     *
+     * <p>Set this property to {@code json} to use the JSON formatter,
+     * or leave it empty to use the legacy plain text formatter.
+     */
+    public static final String CONFIG_DEBUG_LOGFILE_FORMAT = "org.forgerock.openam.debug.format";
+
+    public static final String DEFAULT_DEBUG_SUFFIX_FORMAT = "-yyyy-MM-dd'T'HH-mm-ss.SSSXX";
```

Comment:
I don't think this change is necessary. If we want to change the filename suffix, I would omit the fraction and timezone from this pattern.